src/PortingAssistantExtensionClientShared/Commands/SolutionAssessmentCommand.cs

```diff
@@ -100,15 +100,16 @@ private async void Execute(object sender, EventArgs e)
             {
                 if (!await CommandsCommon.CheckLanguageServerStatusAsync()) return;
```

Comment:
also could you check if the sequence is correct for SolutionPortingCommand and projectportingCommand?

Reply:
There's a semantic check. The format of pull request commit message should be "fix: Resolve UI issues in target framework dialog". Please refer to this doc: https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md